website/project/model.py

```diff
@@ -2311,6 +2311,11 @@ def remove_contributors(self, contributors, auth=None, log=True, save=False):
                 contributor=contrib, auth=auth, log=False,
             )
             results.append(outcome)
+            if outcome:
+                for file_checked in contrib.files_checked_out():
+                    if file_checked.node == self:
```

Comment:
Comparing the nodes by their ids would be more reliable here than comparing the objects.